app/routes/changes-enquiries/deferral/functions.js

```diff
@@ -0,0 +1,124 @@
+const i18n = require('i18next');
+const request = require('request-promise');
+
+const dataStore = require('../../../../lib/dataStore');
+const dateHelper = require('../../../../lib/dateHelper');
+const deferralObject = require('../../../../lib/objects/deferralObject');
+const deleteSession = require('../../../../lib/deleteSession');
+const formValidator = require('../../../../lib/formValidator');
```

Comment:
Could you split out all the deferral validation into its own file, I've started doing this over this over the last year as that `formValidator` file is getting far too big

Reply:
done